### PR TITLE
Use ES6 syntax.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: navbar-component.js [hunk 1]
```diff
@@ -93,14 +87,16 @@ function Ctrl(
...
+          '" is not a valid slot in the navbar.');
       }
       var stack = _stack[slot] = _stack[slot] || [];
       target = angular.element(target[0]);
 
       // clean up element when its scope is destroyed
       options.scope.$on(
-        '$destroy', _destroy(element, options.operation, target, stack));
+        '$destroy', _destroy(element, options.operation, target, stack)
+      );
```
Comment (dlongley) [inline navbar-component.js]: What's the reason for this change?

Comment (dlongley) [inline navbar-component.js]: Does this need to be changed?

**Review by dlongley — CHANGES_REQUESTED** [review]

**Review by davidlehn — APPROVED** [review]

**Comment by mattcollier** [comment]
@dlongley other issues resolved.  Waiting to hear from @davidlehn about forge.

**Comment by mattcollier** [comment]
@dlongley ready for another pass

File: navbar-component.js [hunk 2]
```diff
@@ -93,7 +87,8 @@ function Ctrl(
 
       var target = $element.find('[br-slot="' + slot + '"]');
       if(target.length === 0) {
-        throw new Error('"' + slot + '" is not a valid slot in the navbar.');
+        throw new Error('"' + slot +
+          '" is not a valid slot in the navbar.');
```
Comment (dlongley) [inline navbar-component.js]: Is this an 80 char break?

**Review by dlongley — CHANGES_REQUESTED** [review]

File: navbar-component.js (hunk 2 above)
Comment (mattcollier) [reply navbar-component.js]: Fixed.

**Review by dlongley — APPROVED** [review]
